gcexport.py

```diff
@@ -1132,6 +1143,56 @@ def copy_details_to_summary(summary, details):
     summary['elevationCorrected'] = details['metadataDTO']['elevationCorrected'] if 'metadataDTO' in details and 'elevationCorrected' in details['metadataDTO'] else None
 
 
+class BitWarden:
```

Comment:
I wonder if we should start making separate source files for this kind of classes?

Reply:
Good question!
I thought the same way, but now I believe many users appreciate that this is a _single file_ script so that it can be put in different locations like Garmin activities, scripts path, etc.

It could be wise to split the file into a few smaller ones for ease of maintenance, but I think this is overengineering at the moment. I would wait for such a decision until we really see the real benefits of it or the current design starts slowing down the development velocity.

What are your thoughts, @pe-st?

Reply:
It's not really a *single file* script any more, it needs also the `filtering.py` and one of the `csv_header_*.properties` files. That's the reason that I make zip file releases. So if separate files make life easier, I'm not against.

My goal is that the zip distribution is self contained and easy to use, i.e. the script works without any further manipulations like installing resources with pip etc

Reply:
Oh, good point! I don't know why I forgot about them ;)
I will add a new module for password managers.